spec/lib/fields/has_many_spec.rb

```diff
@@ -1,6 +1,7 @@
 require "administrate/field/has_many"
 require "support/constant_helpers"
 require "support/mock_relation"
+require "rails_helper"
```

Comment:
Apparently for some reason when running only one file of the spec. Will remove 👍